### PR TITLE
20210909 custom deployment release notes

### PR DESCRIPTION
**Pull request by alroba**
New topic for Custom Deployment Documentation:
- Release Notes

cc @pablomoniz

### CODE REVIEW

File: app/content/deployment-options/custom-deployment/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Release Notes
+
+### 21.08.19-2
+* PostgreSQL Data Base upgraded to version 12.7
```
Comment (borja-munoz) [inline app/content/deployment-options/custom-deployment/release-notes.md:4]: database is more common than data base

File: app/content/deployment-options/custom-deployment/release-notes.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+* Ability to disable Diagnosis endpoint
+* Fix ArcGIS import issue when there are no features
+* Support for latest HERE API key format
+* General improvements and performance optimizations
+
+### 21.06.14
+* First stable version
```
Comment (borja-munoz) [inline app/content/deployment-options/custom-deployment/release-notes.md:11]: This is not the first stable version for the on-prem. We should add more detail like "First stable version based on Docker".

File: app/content/deployment-options/custom-deployment/release-notes.md (hunk 1 above)
Comment (alroba) [reply app/content/deployment-options/custom-deployment/release-notes.md]: Perfect, change done: `Data Base` -> `DataBase` 😁

File: app/content/deployment-options/custom-deployment/release-notes.md (hunk 2 above)
Comment (alroba) [reply app/content/deployment-options/custom-deployment/release-notes.md]: Clarification added!

Thanks!

**Review by borja-munoz — APPROVED** [review]

**Pull request merged**